src/UI/Implementation/Component/ViewControl/Renderer.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/* Copyright (c) 2017 Jesús López <lopez@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Implementation\Component\ViewControl;
+
+use ILIAS\UI\Implementation\Render\AbstractComponentRenderer;
+use ILIAS\UI\Renderer as RendererInterface;
+use ILIAS\UI\Component;
+
+/**
+ * Class Renderer
+ * @package ILIAS\UI\Implementation\Component\ViewControl
+ */
+class Renderer extends AbstractComponentRenderer
+{
+	const MODE_ARIA = "Mode View Controler";
```

Comment:
@xus see, e.g. 
$tpl->setVariable("LABEL", $this->txt($component->getAriaLabel()));
in src/UI/Implementation/Component/Glyph/Renderer.php